doc/releases/changelog-dev.md

```diff
@@ -292,8 +273,8 @@
 
 * Reduce usage of `MeasurementProcess.return_type`. Use `isinstance` checks instead.
   [(#3399)](https://github.com/PennyLaneAI/pennylane/pull/3399)
-  
-<h4>Return types project</h4>
+
+<h4>(Experimental) Return types project</h4>
```

Comment:
@Jaybsoni Are most of these entries kinda the same as what was in v0.27? 🤔 
![Screen Shot 2022-12-07 at 11 50 25 AM](https://user-images.githubusercontent.com/34751083/206240475-f77d5517-0c98-423b-b3c3-cf7ccf684fec.png)

Reply:
They're different PRs, but maybe I just need a bit more clarity on what the main differences are between the return types additions in v0.27 and v0.28

Reply:
For this I think asking @antalszava and @rmoyard is probably the best way to go about it.

Reply:
Are these features in this release continuations and further developments of things that were started in v0.27?

Reply:
This release we add support for Pytorch interfaces, Jax-jit interface and also all the batch_transforms. We aded support for shot vectors as well. I think it might be mixed in the releases as I don't see everything here.